tools/bin/release_version.sh

```diff
@@ -25,8 +25,8 @@ GIT_REVISION=$(git rev-parse HEAD)
 
 echo "Building and publishing version $VERSION for git revision $GIT_REVISION..."
 ./gradlew clean composeBuild
-GIT_REVISION=$GIT_REVISION docker-compose -f docker-compose.build.yaml -f docker-compose.yaml build
-GIT_REVISION=$GIT_REVISION docker-compose -f docker-compose.build.yaml -f docker-compose.yaml push
+GIT_REVISION=$GIT_REVISION docker-compose -f docker-compose.build.yaml -f docker-compose.push.yaml -f docker-compose.yaml build
```

Comment:
Do you remember why we need to have `docker-compose.yaml` when we do the build/push? What would happened if we only had build and push ?

Reply:
i don't know why it's there. i was just experimenting with removing it and it works fine so i am going to do so.